content/source/assets/stylesheets/_docs.scss

```diff
@@ -37,7 +37,7 @@
 
     margin-top: 20px; // fallback for non-grid browsers
     @supports (display: grid) {
-      margin-top: 0;
+      // margin-top: 0; // commented out to account for two row headers
```

Comment:
The right way to do this is to remove these whole `@supports` query blocks, and also remove the "only for non-grid" comments above. But I approve the general approach of just leaving some extra space. 👍🏼

Reply:
yep, i would have removed all the unnecessary code, but wasn't sure if we were going to need it in the future, seeing how this is just a temp fix 

```
 // fallback for non-grid browsers
    @supports (display: grid) {
      // margin-top: 0; // commented out to account for two row headers
    }
```